Meter.h

```diff
@@ -64,6 +64,7 @@ typedef struct MeterClass_ {
    const Meter_GetCaption getCaption;
    const Meter_GetUiName getUiName;
    const int defaultMode;
+   const int supportedMode;
```

Comment:
Pleas add a comment to indicate this field does not hold literal values but bit-shifted ones.